components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -599,6 +600,12 @@ struct AccountInfo {
   string keyring_id;
 };
 
+struct AllAccountsInfo {
+  array<AccountInfo> accounts;
+  // TODO(apaymyshev): support selected index
+  // uint32 selected_index;
```

Comment:
This will be done in my next PR where per-coin selected accounts should be a dApp only thing, but ui(both desktop and android) should have a consistent way to get list of all accounts and of them selected.